main.py

```diff
@@ -36,41 +34,42 @@ def calculate_anisotropy_field():
     return -spins_array[atom].spin_position_z*spins_array[atom].spin_position_z*spins_array[atom].anisotropy
```

Comment:
Could we clarify which data this function mutates by passing it in? Or having this logic as method on an object?